docs/index.rst

```diff
@@ -0,0 +1,16 @@
+===========================================
+Welcome to the documentation of red-queen
+===========================================
+
+.. toctree::
+
+    config
+    api
```

Comment:
You can remove this for now since we don't have any contents in the docs yet. These are table of contents (toc) links to other documents which don't exist

Reply:
Consider it done.